README.md

```diff
@@ -21,6 +21,12 @@ kind get kubeconfig > tmp/kubeconfig
 export KUBECONFIG="$PWD/tmp/kubeconfig"
```

Comment:
By [default](https://github.com/sustainable-computing-io/kepler-operator/blob/fea9b1905782773895fab5f575747a2bb11951a1/hack/cluster-up.sh#L25) it will setup kind cluster